File: 프로그래머스/0/120896. 한 번만 등장한 문자/한 번만 등장한 문자.java
import java.util.*;
class Solution {
    public String solution(String s) {
        // 결과를 저장할 
        StringBuilder answer = new StringBuilder();
        
        // 한 번만 등장하는 문자 찾기
        for (char c = 'a'; c <= 'z'; c++) {
            if (s.indexOf(c) != -1 && s.indexOf(c) == s.lastIndexOf(c)) {
                answer.append(c);
            }
        }

        return answer.toString();
    }
}

File: 프로그래머스/0/120896. 한 번만 등장한 문자/한 번만 등장한 문자.js
function solution(s) {
    // 결과를 저장할 문자열 배열
    let answer = [];
    
    // 한 번만 등장하는 문자 찾기
    for (let c = 'a'.charCodeAt(0); c <= 'z'.charCodeAt(0); c++) {
        let char = String.fromCharCode(c);
        if (s.indexOf(char) !== -1 && s.indexOf(char) === s.lastIndexOf(char)) {
            answer.push(char);
        }
    }
    
    return answer.join('');
}
